rpm/0002-Add-Triple-isMeeGo.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ruben De Smet <ruben.de.smet@rubdos.be>
+Date: Sun, 26 Jun 2022 10:41:44 +0200
+Subject: [PATCH] Add Triple::isMeeGo()
+
+---
+ llvm/include/llvm/ADT/Triple.h | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/llvm/include/llvm/ADT/Triple.h b/llvm/include/llvm/ADT/Triple.h
+index 9e36dfd4d2d5..8e66888615ff 100644
+--- a/llvm/include/llvm/ADT/Triple.h
++++ b/llvm/include/llvm/ADT/Triple.h
+@@ -657,6 +657,9 @@ public:
+   /// Tests whether the target is Android
+   bool isAndroid() const { return getEnvironment() == Triple::Android; }
+ 
++  /// Tests whether the target is Android
```

Comment:
Suppose this shouldn't talk about android if it's isMeego. Not sure if that's a good naming, but whatever gets this built.

Reply:
Fixed the comment